app/include/dt-bindings/zmk/keys.h

```diff
@@ -4,146 +4,161 @@
+#define SPC 0x2C    // Keyboard Space
+#define MINUS 0x2D  // Keyboard - and _ (Minus and Underscore) 
+#define EQL 0x2E    // Keyboard = and + (Equals and Plus)
+#define LBKT 0x2F   // Keyboard [ and { (Left Bracket and Left Curly Bracket)
+#define RBKT 0x30   // Keyboard ] and } (Right Bracket and Right Curly Bracket)
+#define BSLH 0x31   // Keyboard \ and | (Backslash and Pipe)
+#define TILD 0x32   // Keyboard Non-US # and ~ (Pound/Hash/Number and Tilde)
+#define SCLN 0x33   // Keyboard ; and : (Semicolon and Colon)
+#define QUOT 0x34   // Keyboard ' and " (Appostraphy and Quote)
+#define GRAV 0x35   // Keyboard ` and ~ (Grave/Backtick and Tilde)
+#define CMMA 0x36   // Keyboard , and < (Comma and Less Than) 
+#define DOT  0x37   // Keyboard . and > (Period and Greater Than)
+#define FSLH 0x38   // Keyboard / and ? (Forward Slash and Question)
+#define CLCK 0x39   // Keyboard Caps Lock
+#define F1 0x3A // Keyboard F1
+#define F2 0x3B // Keyboard F2
+#define F3 0x3C // Keyboard F3
+#define F4 0x3D // Keyboard F4
+#define F5 0x3E // Keyboard F5
+#define F6 0x3F // Keyboard F6
+#define F7 0x40 // Keyboard F7
+#define F8 0x41 // Keyboard F8
+#define F9 0x42 // Keyboard F9
+#define F10 0x43    // Keyboard F10
+#define F11 0x44    // Keyboard F11
+#define F12 0x45    // Keyboard F12
+
+#define PRSC 0x46   // Keyboard Print Screen
+#define SCLK 0x47   // Keyboard Scroll Lock
+#define PAUS 0x48   // Keyboard Pause/Break
+#define INS 0x49    // Keyboard Insert
+#define HOME 0x4A   // Keyboard Home
+#define PGUP 0x4B   // Keyboard Page Up
+#define DEL  0x4C   // Keyboard Delete
+#define END  0x4D   // Keyboard End
+#define PGDN 0x4E   // Keyboard Page Down
+#define RARW 0x4F   // Keyboard Right Arrow
+#define LARW 0x50   // Keyboard Left Arrow
+#define DARW 0x51   // Keyboard Down Arrow 
+#define UARW 0x52   // Keyboard Up Arrow
+
+#define KDIV 0x54   // Keypad / (Divide)
+#define KMLT 0x55   // Keypad * (Multiply)
+#define KMIN 0x56   // Keypad - (Minus)
+#define KPLS 0x57   // Keypad + (Plus)
+
+#define GUI 0x65    // Keyboard GUI (Windows/Command)
+
+#define UNDO 0x7A   // Keyboard Undo
+#define CUT 0x7B    // Keyboard Cut
+#define COPY 0x7C   // Keyboard Copy
+#define PSTE 0x7D   // Keyboard Paste
+
+#define CURU 0xB4   // Keyboard Currency Unit
+
+#define LPRN 0xB6   // Keypad ( (Left Parenthesis)
+#define RPRN 0xB7   // Keypad ) (Right Parenthesis)
+#define LCUR 0xB8   // Keypad { (Left Curly Bracket)
+#define RCUR 0xB9   // Keypad } (Right Curly Bracket)
+
+#define CRRT 0xC3   // Keypad ^ (Caret)
+#define PRCT 0xC4   // Keypad % (Percent)
+#define LABT 0xC5   // Keypad > (Greater Than)
+#define RABT 0xC6   // Keypad < (Less Than)
+#define AMPS 0xC7   // Keypad & (Ampersand)
+#define PIPE 0xC9   // Keypad | (Pipe)
+#define COLN 0xCB   // Keypad : (Colon)
+#define HASH 0xCC   // Keypad # (Pound/Hash/Number)
+#define KSPC 0xCD   // Keypad Space
+#define ATSN 0xCE   // Keypad @ (At Sign)
+#define BANG 0xCF   // Keypad ! (Exclamation)
```

Comment:
For quite a few of these, do we want them there if they don't do anything (or at least that is my current understanding).

Perhaps there should be a block comment in this file with "These symbols can be achieved with X syntax instead, and are ommited here due to changing  for different OS keyboard layouts, see this doc: ...".

Then we can list all these symbols there so if a user does a search in this file for "Exclamation" they get a link to the upcoming shifted docs instead of being confused why its missing.